AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/sample.json

```diff
@@ -5,7 +5,7 @@
   "description": "This sample illustrates how to train a TensorFlow model and run inference with oneMKL and oneDNN.",
   "builder": ["cli"],
   "languages": [{"python":{}}],
-  "os":["linux"],
+  "os":["linux",, "windows"],
```

Comment:
I see two commas here. Can you please validate sample.json?

Reply:
I have updated the file. Please help to review.  thanks